### DISCUSSION TITLE
Allow using DEL key to delete a selection in the Hex Editor instead of 4 actions

### DISCUSSION THREAD

**Discussion by shazz** [WerWolv/ImHex#1885] [Feedback]
Right now:
- Select with the mouse the set of bytes
- Right click
- Remove
- Check address and size
- Click set

**Comment by WerWolv** [comment]
It's by design that resizing actions take extra steps. It's to avoid accidentally resizing the file (which is in most cases not intentional and actually destroys the file's format). Therefor adding a single key-press to do it is not going to happen. 

I agree though we should probably skip the popup if there's something selected already.

**Reply by shazz** [reply]
yeah I got, it was by design for safety and not important, I'm just lazy as I spend some time to remove useless bytes from files to only keep why I need.

And yes the popup is somewhat duplicate as the selection did the job.

Thanks for watching ;)

**Reply by Haarolean** [reply]
One could have a setting to do auto-resize or whatever, maybe users are not that stupid to not allow them to delete the bytes
